packages/shell-api/src/collection.ts

```diff
@@ -2076,6 +2076,44 @@ export default class Collection extends ShellApiWithMongoClass {
     });
   }
 
+  /**
+   * Helper for getting collection info of sharded timeseries collections
+   * @returns collection info based on given collStats, null if the information is not found or
+   * if the collection is not timeseries.
+   */
+  async _getShardedTimeseriesCollectionInfo(
+    config: Database,
+    collStats: Document[]
+  ): Promise<Document | null> {
+    const timeseriesShardStats = collStats.find(
+      (extractedShardStats) =>
+        typeof extractedShardStats.storageStats.timeseries !== 'undefined'
+    );
+
+    if (!timeseriesShardStats) {
+      return null;
+    }
+
+    const { storageStats } = timeseriesShardStats;
+
+    const timeseries: Document = storageStats['timeseries'];
+
+    const timeseriesBucketNs: string | undefined = timeseries['bucketsNs'];
```

Comment:
I'm a bit surprised the linter doesn't say anything, but either way, we can just use regular JS property accesses here, right?

```suggestion
    const timeseries: Document = storageStats.timeseries;

    const timeseriesBucketNs: string | undefined = timeseries.bucketsNs;
```